entities/Natural/Planet.cs

```diff
@@ -8,15 +8,40 @@ public class Planet : Desired
         public AtmosphereType atmosphere;
         public GravityType gravity;
         public TemperatureType temperature;
-        public WaterType water;
-        public Planet(string name, int size, int atmosphere, int gravity, int temperature, int water)
+        public WaterResourceType water;
+        public FloraType flora;
+        public FaunaType fauna;
+        public CivilizationType civilization;
+        public Planet(string name, IDictionary<string,int> parameters)
```

Comment:
Klucz nie jako string tylko jako enum, wtedy nie będziesz musiał tyle ifować

Reply:
Jak to zrobić? Nie mogę wrzucić typu 'Enum', a każdy z nich to osobny typ.